machina/apps/forum_permission/shortcuts.py

```diff
@@ -17,42 +17,76 @@
 GroupForumPermission = get_model('forum_permission', 'GroupForumPermission')
 UserForumPermission = get_model('forum_permission', 'UserForumPermission')
 
+ALL_AUTH = object()  # object to check against if all authenticated users are meant
```

Comment:
`ALL_AUTH` is not really explicit. What about `ALL_AUTHENTICATED_USERS`?

Reply:
I'll look into it, but the shorter name was mainly to avoid too long lines further along in the code.

Reply:
Fixed this.